src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,56 @@
+import classNames from 'classnames';
+import React from 'react';
+import { filterLinks } from '../../helper/filterLinks';
+import { Filter } from '../../types/Filter';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  filter: Filter,
+  onFilter: (filter: Filter) => void,
+  completedTodos: Todo[],
+  activeTodosCount: number,
+  deleteCompleted: (todos: Todo[]) => void,
+};
+
+export const Footer: React.FC<Props> = React.memo(
+  ({
+    filter,
+    onFilter,
+    completedTodos,
+    activeTodosCount,
+    deleteCompleted,
+  }) => {
+    return (
+      <footer className="todoapp__footer">
+        <span className="todo-count">
+          {`${activeTodosCount} items left`}
+        </span>
+
+        <nav className="filter">
+          {filterLinks.map(link => (
+            <a
+              key={link.title}
+              href={`#/${link.url}`}
+              className={classNames(
+                'filter__link',
+                { selected: filter === link.title },
+              )}
+              onClick={() => onFilter(link.title)}
```

Comment:
To prevent the default behavior of the anchor tag, consider adding `event.preventDefault()` inside the onClick handler 🛑